config/api/src/main/java/org/jboss/windup/config/metadata/LabelMetadataBuilder.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.windup.config.metadata;
+
+public class LabelMetadataBuilder extends AbstractLabelsetMetadata implements LabelProviderMetadata
+{
+    private String origin;
+    private String description;
+    private int priority;
+
+    public LabelMetadataBuilder(String ID, String description) {
+        super(ID);
+        this.description = description;
+        this.priority = Integer.MAX_VALUE;
```

Comment:
we could replace this with `this(ID, description, Integer.MAX_VALUE);`

Reply:
done

Reply:
I meant the other way around to have the constructors to become:
```Java
    public LabelMetadataBuilder(String ID, String description)
    {
        this(ID, description, Integer.MAX_VALUE);
    }

    public LabelMetadataBuilder(String ID, String description, int priority)
    {
        super(ID);
        this.description = description;
        this.priority = priority;
    }
```

Reply:
@mrizzi thanks a lot... it is done :)